cheroot/test/test_server.py

```diff
@@ -236,3 +236,49 @@ def test_peercreds_unix_sock_with_lookup(peercreds_enabled_server):
         peercreds_text_resp = requests.get(unix_base_uri + PEERCRED_TEXTS_URI)
         peercreds_text_resp.raise_for_status()
         assert peercreds_text_resp.text == expected_textcreds
+
+
+def test_high_number_of_file_descriptors():
+    """Test the server does not crash with a high file-descriptor value.
+
+    This test should cause a server crash, as the server will try to
+    select() a file-descriptor higher than 1024.
+    """
+    increased_nofile_limit = 2048
+    sockets = []
+    try:
+        import resource
+        import socket
```

Comment:
plz move these on the top of the test module.

Reply:
Done, and renamed some other variables that were causing issues with the globals.